servicetalk-grpc-gradle-plugin/src/main/groovy/io/servicetalk/grpc/gradle/plugin/ServiceTalkGrpcPlugin.groovy

```diff
@@ -67,6 +62,49 @@ class ServiceTalkGrpcPlugin implements Plugin<Project> {
         throw new InvalidUserDataException("Please set `serviceTalkGrpc.protobufVersion`.")
       }
 
+      // If this project is outside of ServiceTalk's gradle build we need to add an explicit dependency on the
+      // uber jar which contains the protoc logic, as otherwise the grpc-gradle-plugin will only add a dependency
+      // on the executable script
+      File uberJarFile
+      String scriptNamePrefix
+      if (serviceTalkProtocPluginPath) {
+        scriptNamePrefix = "servicetalk-grpc-protoc-" + project.version
+        uberJarFile = new File(serviceTalkProtocPluginPath)
+      } else {
+        scriptNamePrefix = "servicetalk-grpc-protoc-$serviceTalkVersion"
```

Comment:
Why `if` branch should use `project.version` but `else` branch uses `$serviceTalkVersion`? Are they different?

Reply:
> Are they different?

Yes. if `serviceTalkProtocPluginPath` is set then `serviceTalkVersion == ${serviceTalkVersion}` (or something similar). Basically if `serviceTalkProtocPluginPath` is set the variable isn't reliably available and instead we use the current project version (because you are assumed to be building in the same project anyways ... IIUC this should be true even if using `--include-build`)